### ISSUE TITLE
COCO_A test set 68% mIoU, what I am missing?

### ISSUE THREAD

**Issue by JaeLee18** [cvlab-columbia/pix2gestalt#19]
Hello,
Thank you for the amazing work.

I downloaded the pre-trained weight and the COCO-A dataset.
I calculated the mIoU metric over COCO-A test set but I am getting 68% mIoU.

Are there any specific filters I should apply for? such as excluding some objects based on X condition.

**Comment by egeozguroglu** [comment]
Hi, we haven't yet cleaned up full-fledged AmodalCOCO/BSDS eval scripts but instead released user-friendly inference functionality for amodal completion and segmentation, including a Gradio demo. You may not be using, pre-processing [Amodal COCO](https://github.com/Wakeupbuddy/amodalAPI) & Amodal BSDS correctly, or accurately reproducing the evaluation pipeline (e.g. sampling 16 variations, majority voting, etc). Please email me, and we can share specific scripts to reproduce any provided table.

**Comment by imlixinyang** [comment]
Hi @egeozguroglu, can you share the evaluation scripts to imlixinyang@gmail.com. Many thanks for your great work.

**Comment by egeozguroglu** [comment]
> Hi [@egeozguroglu](https://github.com/egeozguroglu), can you share the evaluation scripts to [imlixinyang@gmail.com](mailto:imlixinyang@gmail.com). Many thanks for your great work.

Shared @imlixinyang

**Comment by weikaih04** [comment]
Hi @egeozguroglu, could you also share the evaluation scripts with weikaih@cs.washington.edu for replication? Thanks for the great work!

**Comment by Jason-aplp** [comment]
Hi @egeozguroglu, could you also share the evaluation scripts with [jason_lu@pku.edu.cn](mailto:jason_lu@pku.edu.cn) for replication? Thanks for the great work!
